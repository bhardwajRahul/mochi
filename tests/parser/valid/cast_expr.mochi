 type Todo {
   title: string
 }
 let x = {"title": "a"} as Todo
