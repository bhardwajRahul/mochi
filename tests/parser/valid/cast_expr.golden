(program
  (type Todo
    (field title (type string))
  )
  (let x
    (cast
      (map
        (entry (string title) (string a))
      )
      (type Todo)
    )
  )
)
